muskathlon/models/event_compassion.py

```diff
@@ -8,19 +8,63 @@
 #    The licence is in the file __manifest__.py
 #
 ##############################################################################
-from odoo import models, fields
+from odoo import models, fields, api
+import re
 
 
 class EventCompassion(models.Model):
     _inherit = 'crm.event.compassion'
 
     thank_you_text = fields.Html(translate=True)
-
     muskathlon_event_id = fields.Char(
         string="Muskathlon event ID", size=128)
     muskathlon_registration_ids = fields.One2many(
         'muskathlon.registration', 'event_id', 'Muskathlon registrations')
 
+    public_description = fields.Text('Description publique')
+    picture_1 = fields.Binary('Picture 1')
+    picture_2 = fields.Binary('Picture 2')
+    video_url = fields.Char("Video URL")
+    participants_amount_objective = fields.Integer(
+        'Default raise objective by participant', default=10000, require=True)
+    amount_objective = fields.Integer(readonly=True,
+                                      compute='_compute_amount_raised')
+    amount_raised = fields.Integer(readonly=True,
+                                   compute='_compute_amount_raised')
+    amount_raised_percents = fields.Integer(readonly=True,
+                                            compute='_compute_amount_raised')
+
+    def _compute_amount_raised(self):
+        for event in self:
+            amount_raised = 0
+            amount_objective = 0
+
+            for registration in event.muskathlon_registration_ids:
+                amount_raised += registration.amount_raised
+                amount_objective += registration.amount_objective
+
+            event.amount_raised = amount_raised
+            event.amount_objective = amount_objective
+            event.amount_raised_percents = int(
+                amount_raised * 100 / amount_objective)
+
+    @api.model
+    def getEventParticipants(self, event_id):
```

Comment:
method names should be lowercase with underscores between words

Reply:
Yeah, done.